src/site/content/en/mini-apps/mini-app-super-apps/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Mini apps and super apps
+authors:
+  - thomassteiner
+date: 2021-02-25
+# updated: 2021-02-25
+description: |
+  This chapter introduces the concept of super apps and presents the major players.
+tags:
+  - mini-apps
+---
+
+## Welcome to mini apps
+
+When you look at the applications on your phone, you probably have specific apps for specific tasks.
+You might have a banking app. You might have an app for buying public transit tickets. Likely you
+have an app for getting directions, etc. In this post, I introduce you to the concept of a different
+kind of app, namely mini apps, sometimes also referred to as mini programs or applets. I first
+provide some background on the various mini app platforms and their developer experience, and then
+focus on things the web can learn from mini apps. But before I start writing about mini apps, I
+first need to talk about super apps.
+
+## What are super apps?
+
+Super apps are apps that are hosts to other apps that run within the super app: the so-called mini
+apps. Popular super apps are Tencent's [WeChat](https://weixin.qq.com/) (微信), the app of the
+search engine [Baidu](https://baidu.com/) (百度), Ant Group's (an affiliate company of the Chinese
+Alibaba Group) [Alipay](https://www.alipay.com/) (支付宝), but also ByteDance's
+[Douyin](https://www.douyin.com/) (抖音), which you might know as TikTok (蒂克托克). The first three
+are commonly also referred to as BAT, which derives from **B**(aidu)**A**(libaba)**T**(encent). Super apps have taken the
+Chinese market by storm, which is why a lot of the examples in this article are Chinese.
+
+<figure class="w-figure">
+  {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/UKmUgG231MtQ2nEo1P0K.PNG", alt="List of recently launched mini apps in the WeChat super app.", width=300, height=649 %}
+  <figcaption class="w-figcaption">The WeChat super app showing recently launched mini apps.</figcaption>
+</figure>
+
+### Short characterization of each super app platform
```

Comment:
```suggestion
### A few words about super app platforms
```

Reply:
Just trying out a simpler subtitle, open to other suggestions, eg. "Roles of super app platforms"